crates/dbsp/src/trace/test_batch.rs

```diff
@@ -257,7 +268,56 @@ where
 {
     data: BTreeMap<(K, V, T), R>,
     lower_key_bound: Option<K>,
-    lower_val_bound: Option<V>,
+    #[size_of(skip)]
+    key_filter: Option<Filter<K>>,
+    #[size_of(skip)]
+    value_filter: Option<Filter<V>>,
+}
+
+impl<K, V, T, R> Debug for TestBatch<K, V, T, R>
+where
+    K: Ord + Debug,
+    V: Ord + Debug,
+    T: Ord + Debug,
+    R: Debug,
+{
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
```

Comment:
does this belong in test code or should it be in a more useful place?

Reply:
This is for `TestBatch` only, so doesn't really apply elsewhere.

Reply:
seemed to be handy to have a Debug method for filters that prints them in this way